packages/queued-request-controller/src/QueuedRequestController.ts

```diff
@@ -83,6 +83,7 @@ export type QueuedRequestControllerOptions = {
     request: QueuedRequestMiddlewareJsonRpcRequest,
   ) => boolean;
   clearPendingConfirmations: () => void;
+  showApprovalRequest: () => void;
```

Comment:
should this be optional?..

Reply:
probably not? this controller will only be used on the extension AFAIK and I believe we will always want this hook passed?

Reply:
it's optional in the sense that this isn't required for the QueuedRequestController to perform it's core function

Reply:
sure thats definitely true. I guess I think of the type as safeguard for accidental removal... though maybe that's not really true